filesystem/trd.ksy

```diff
@@ -0,0 +1,138 @@
+meta:
+  id: trd
```

Comment:
KS file formats collection does not have any namespacing — basically, everything eventually gets dumped into a single bin, and the only way you can reference this format is by using "trd". So, we typically strive very hard to not use abbreviations and other short ambiguous IDs. In this particular case, I can't be sure that in future we won't have a clash with some other "trd" — especially given that [Wikipedia gives us so many alternatives](https://en.wikipedia.org/wiki/TRD).

Can I ask you to use a longer name (like `tr_dos_image` or something like that)?

Reply:
Changed to `tr_dos_image`.